crates/turborepo-cache/src/cache_archive/restore.rs

```diff
@@ -17,12 +17,15 @@ use crate::{
 };
 
 pub struct CacheReader<'a> {
-    reader: Box<dyn Read + 'a>,
+    reader: Box<dyn Read + Send + 'a>,
```

Comment:
Pretty sure this is no longer needed
```suggestion
    reader: Box<dyn Read + 'a>,
```